Doc/library/unittest.mock.rst

```diff
@@ -1078,6 +1080,37 @@ object::
       [call('foo'), call('bar')]
 
 
+.. class:: ThreadingMock(spec=None, side_effect=None, return_value=DEFAULT, wraps=None, name=None, spec_set=None, unsafe=False, **kwargs)
```

Comment:
Threaded rather than Threading maybe?

Reply:
Happy to look for a better name. We discussed EventMock as well in the sprints, but could not find a better name. I'm not a native English speaker, if ThreadedMock "is more correct", happy to rename it.

Reply:
I guess it depends what it is, if it's a mock that helps when dealing with threading, then ThreadingMock feels right. If it's a mock that is threaded, then ThreadedMock instead. I'm afraid I don't understand the use case enough to give you a definitive answer.

Reply:
It is the former, so I guess the naming is fine :). Thanks!